Website not building

@mairin if you have any sage advice, I would appreciate it.  I'm not able to gronk out where the error is, that's foiling the builds that have updates to the community section of the site.  For instance:  https://github.com/containers/podman.io/pull/276

I'm seeing a slightly more detailed error when I try to build the site on my machine, but not detailed enough for me to figure out what's going wrong.  It seems that docusaurus is upset with something, but I'm not sure what.  Thoughts or pointers for further investigation?

Error on build:

```
yarn build
[INFO] [en] Creating an optimized production build...

✔ Client
  

✖ Server
  Compiled with some errors in 1.38m

[ERROR] Docusaurus server-side rendering could not render static page with path /community.


TypeError: Cannot read properties of undefined (reading '0')
    at Array.forEach (<anonymous>)
    at Array.forEach (<anonymous>)
[ERROR] Unable to build website for locale en.
[ERROR] Error: Failed to compile with errors.
    at /root/tsweeney/workspaces/containers/podman.io/node_modules/@docusaurus/core/lib/webpack/utils.js:180:24
    at /root/tsweeney/workspaces/containers/podman.io/node_modules/webpack/lib/MultiCompiler.js:554:14
    at processQueueWorker (/root/tsweeney/workspaces/containers/podman.io/node_modules/webpack/lib/MultiCompiler.js:491:6)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
[INFO] Docusaurus version: 2.4.3
Node version: v16.18.1
```

--- a/src/components/layout/CommunityMeetingsCardGrid/index.tsx
+++ b/src/components/layout/CommunityMeetingsCardGrid/index.tsx
@@ -93,8 +93,8 @@ function CommunityMeetingsCardGrid({ cards }) {
     Object.values(markDownFiles)?.forEach(mdFile => {
       let mdReader = mdFile?.default(useRef());
       mdReader?.props?.children?.forEach(child => {
-        let field1: string = child?.props?.children[0];
-        let field2: object = child?.props?.children[1];
+        let field1: string = child?.props?.children?.[0];
+        let field2: object = child?.props?.children?.[1];
         if (typeof field1 == 'string' && (field1.includes('BlueJeans') || field1.includes('Video'))) {
           if (mdFile?.contentTitle?.includes('Cabal')) {
             cabalDropdownOptions.unshift({

